src/plugins/intel_gpu/src/kernel_selector/kernels/quantize/quantize_kernel_scale_shift_vload8_opt.cpp

```diff
@@ -0,0 +1,150 @@
+// Copyright (C) 2018-2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "quantize_kernel_scale_shift_vload8_opt.h"
+
+#include <iostream>
+#include <string>
+
+#include "kernel_selector_utils.h"
+
+static const size_t vec_size = 8;
+
+namespace kernel_selector {
+ParamsKey QuantizeKernelScaleShift_vload8::GetSupportedKey() const {
+    ParamsKey k;
+    k.EnableInputDataType(Datatype::UINT8);
+    k.EnableInputDataType(Datatype::INT8);
+    k.EnableInputDataType(Datatype::F16);
+    k.EnableInputDataType(Datatype::F32);
+    k.EnableOutputDataType(Datatype::F16);
+    k.EnableOutputDataType(Datatype::F32);
+    k.EnableOutputDataType(Datatype::UINT8);
+    k.EnableOutputDataType(Datatype::INT8);
+    k.EnableAllInputLayout();
+    k.EnableAllOutputLayout();
+    k.EnableTensorPitches();
+    k.EnableBatching();
+    k.EnableDifferentTypes();
+    k.EnableQuantizeScaleShiftOpt();
+    return k;
+}
+
+auto parse_block_size = [](int index, DataLayout dl) {
+    std::string format_str = toString(dl);
+    auto get_block_size = [&] (std::string substr) {
+        auto start_pos = format_str.find(substr);
```

Comment:
Maybe we can list here all the supported formats to be more verbose instead of parsing the format string? Like this:
https://github.com/openvinotoolkit/openvino/blob/bd91467d9c0c24013fd33d671c84f112d51b7b52/src/plugins/intel_gpu/src/kernel_selector/kernels/eltwise/eltwise_kernel_blocked_opt.cpp#L460-L467

This also can be reused in Validate function probably